WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1814,4 +1815,26 @@ FileTypeImage       = @FileIconCommonStyle@ background-color:#50816e;
 FileTypeCal         = @FileIconCommonStyle@ background-color:#94385d;
 FileTypeVCard       = @FileIconCommonStyle@ background-color:#666666;
 FileTypeMedia       = @FileIconCommonStyle@ background-color:#9e61ca;
-FileTypeMail        = @FileIconCommonStyle@ background-color:@PrimaryBrandingColor@
\ No newline at end of file
+FileTypeMail        = @FileIconCommonStyle@ background-color:@PrimaryBrandingColor@
+
+#######################################################################
+#   Contact view styles
+#######################################################################
+
+ContactViewContainer        = @AppContainerBg@ @PanelBorder@; border-width:1px 0 0 0; overflow:auto; box-sizing: border-box;
+
+# contact list view
+ContactListGutterSpace      = 12px
```

Comment:
No, I don't think so, we should have one variable for all spacing. App wise, it can be different.
Probably,  I should change to `ContactListGutterSpace  = @MailViewGutterSpace@`, in case there is app specific changes, it will handle that as well. 

Let me know.